### ISSUE TITLE
web sockets 

### ISSUE THREAD

**Issue by joeblew99** [b3ntly/vue-grpc#4]
The new grpc-web supports web sockets. I dont think this does right ?

**Comment by beeekind** [comment]
@joeblew99 Are you sure? Admittedly I only use this for unary RPC. Can you link me to any relevant docs?

Regardless this project doesn't implement it - though I would happy to throw together a chatroom type example if client-side streaming has been implemented with ws.

**Comment by joeblew99** [comment]
Hey.

Of yeah it works..

https://github.com/johanbrandhorst/grpcweb-example

There is a make file and it takes 2 minutes to get the example going.... 

When you get it going and loaded up in Chrome, there is one page of an example. The BookChat IS using websockets.  
This is currently using React ad gopherjs, but i think you can take alot of this and maek it work with vuejs too for sure.

Come back to me if you have any questions.. I like React and Vuejs... 

BTw the interesting thing about the example is that there is no store or actions or reflux stuff. Each component is self contained and just gets and sets data or gets updates via the websockets.
I am using it with a golang service installed on the desktop or mobile that has a boltDB. This means that i dont really need a reflux, action, store etc in the Web layer because all state is held in the boltDB. I actually like it this way ...

**Comment by beeekind** [comment]
Agreed - redux can feel like a lot of boiler plate especially if you're doing heavy state management work on the server as well.  I much prefer the vuex approach though I can't deny the advantage of a well layed out store module for large SPAs.
